src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java

```diff
@@ -121,7 +121,26 @@ public String getColumnValue(final VariantContext variant) {
 
     AS_RAW_MQRankSum { // TODO -- maybe rely on 1/1 for call_GT, also get rid of the | at the beginning
         public String getColumnValue(final VariantContext variant) {
+            // e.g. AS_RAW_MQRankSum=|1.4,1|NaN;
             String out =  getAttribute(variant, GATKVCFConstants.AS_RAW_MAP_QUAL_RANK_SUM_KEY, null);
+
+            if (out == null) {
+                // Try to use non-AS version
+                // TODO: it looks like the AS_RAW version also trims to a single decimal point??
+                // e.g. MQRankSum=1.465 and turn it into |-0.511,1|
+                String outNotAlleleSpecific = getAttribute(variant, GATKVCFConstants.READ_POS_RANK_SUM_KEY, null);
+
+                if (variant.getAlleles().size() == 3) { // GT 0/1 1/1
+                    out = "|" + outNotAlleleSpecific + ",1|";
+
+                } else if (variant.getAlleles().size() == 4) { // GT 1/2
+                    // TODO: just replicate rather than distribute, is this right?
```

Comment:
I think this is close enough. I tended to distribute when I did this for other metrics, but I don't think it matters and we don't want half of a read in the output anyway :)